config/manager/manager.yaml

```diff
@@ -24,6 +24,7 @@ spec:
             - --leader-elect
             - "--metrics-bind-addr=127.0.0.1:8080"
             - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},AKS=${EXP_AKS:=false}"
+            - "--v=2"
```

Comment:
was this checked in by mistake?

Reply:
I increased verbosity of the test logging. To make it easier, I add an explicit verbosity to the controller.

Reply:
isn't 1 the default though? if so this increases verbosity for all default clusterctl deployments

Reply:
[Looks like the default is 0.](https://pkg.go.dev/k8s.io/klog#pkg-overview) I did not intend on increasing the default. I'll change it.